src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -462,14 +462,13 @@ public void parse_repoAliases_sameResult() throws ParseException, HelpScreenExce
         Assert.assertEquals(repoAliasCliArguments, reposAliasCliArguments);
     }
 
-    @Test
-    public void parse_invalidRepoLocation_emptyRepoConfigurationList()
+    @Test (expected = InvalidLocationException.class)
```

Comment:
I modified the test to expect an exception thrown instead of testing for empty `repoConfigs` list. I guess it's this method causing the `codecov` check to fail? But I'm not sure where does the `-<0.1%` come from. May I know how I should figure it out and fix this?

Reply:
It is ok. Extremely small fluctuations in the code coverage may not be meaningful.

Reply:
I believe this is due to the extra condition added and no test case covering it.

Reply:
Thank you for clarifying!